projects/views.py

```diff
@@ -20,10 +20,12 @@ def feature_view(request):
 
 @login_required
 def project_view(request):
-    data = Project.objects.all()
+    data_p = Project.objects.all()
+    data_f = Feature.objects.all()
```

Comment:
Instead of getting all the features from the database (unrelated to projects), I believe the features for a project are automatically linked. Something like:

`project.feature_set`